### ISSUE TITLE
Authentication using Tauri

### ISSUE THREAD

**Issue by peterpeterparker** [dfinity/internet-identity#2477]
If that's even possible, I would love to read a tutorial and sample code that explains how to integrate II with Tauri. And if such integration is not possible currently, I would just love to be able to integrate II into a Tauri project someday.

Not that I have any particular plan or real project; I just think it would be really cool.

**Comment by aterga** [comment]
Closing — this was an aspirational feature request with no engagement. Tauri integration would be a community SDK effort rather than an II core issue.

**Comment by peterpeterparker** [comment]
Right, I forgot about it. Without using II, I was able to develop an OpenId based proof of concept that used Ionic for a mobile app, maybe I can do the same with Tauri if I ever need it.
